dataspace-connector/full/docker-compose.yml

```diff
@@ -56,9 +56,31 @@ services:
       - configmanager
     restart: always
 
+  keycloak:
+    depends_on:
+      - postgres
+    container_name: local_keycloak
+    environment:

```

Comment:
Please build an external .env-file for this, maybe in the configmanager folder as an additional keycloack.env

Reply:
Done